src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="index.html"
+          class="logo"
+        >
+          <img
+            src="./images/logo.png"
+            alt="Moyo logo"
+            class="logo__img"
+          />
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
```

Comment:
The `href` attribute is set to '#', which is a placeholder. Consider providing a meaningful URL or using JavaScript to handle the click event if this is meant to be a dynamic link.